app/src/actions/queryActions.test.js

```diff
@@ -18,103 +18,304 @@ describe('query actions', () => {
-      "resourceversion": "0",
-      "uid": "0x13d7"
-    }],
-    "clusterip": "100.68.144.37",
-    "k8sobj": "K8sObj",
-    "labels": "{\"app\":\"helm-chart\",\"chart\":\"helm-chart-0.1.0\",\"heritage\":\"Tiller\",\"release\":\"profile-testing\"}",
-    "name": "profile-testing-helm-chart",
-    "namespace": [{
-      "k8sobj": "K8sObj",
-      "labels": "null",
-      "name": "profile-testing",
-      "objtype": "Namespace",
-      "resourceid": "preprod-west2.cluster.k8s.local:profile-testing",
-      "resourceversion": "0",
-      "uid": "0x1e84"
-    }],
-    "objtype": "Service",
-    "ports": "[{\"nodePort\":32313,\"port\":80,\"protocol\":\"TCP\",\"targetPort\":8080}]",
-    "resourceid": "preprod-west2.cluster.k8s.local:profile-testing:profile-testing-helm-chart",
-    "resourceversion": "3574724",
-    "selector": "{\"app\":\"mysqlserver\",\"release\":\"profile-testing\"}",
-    "servicetype": "NodePort",
-    "uid": "0x4c05"
-  },{
-    "cluster": [{
-      "k8sobj": "K8sObj",
-      "name": "preprod-west2.cluster.k8s.local",
-      "objtype": "Cluster",
-      "resourceid": "preprod-west2.cluster.k8s.local",
-      "resourceversion": "0",
-      "uid": "0x13d7"
-    }],
-    "clusterip": "100.68.144.37",
-    "k8sobj": "K8sObj",
-    "labels": "{\"app\":\"helm-chart\",\"chart\":\"helm-chart-0.1.0\",\"heritage\":\"Tiller\",\"release\":\"profile-testing\"}",
-    "name": "profile-testing-helm-chart",
-    "namespace": [{
-      "k8sobj": "K8sObj",
-      "labels": "null",
-      "name": "profile-testing",
-      "objtype": "Namespace",
-      "resourceid": "preprod-west2.cluster.k8s.local:profile-testing",
-      "resourceversion": "0",
-      "uid": "0x1e84"
-    }],
-    "objtype": "Service",
-    "ports": "[{\"nodePort\":32313,\"port\":80,\"protocol\":\"TCP\",\"targetPort\":8080}]",
-    "resourceid": "preprod-west2.cluster.k8s.local:profile-testing:profile-testing-helm-chart",
-    "resourceversion": "3574724",
-    "selector": "{\"app\":\"mysqlserver\",\"release\":\"profile-testing\"}",
-    "servicetype": "NodePort",
-    "uid": "0x4c05"
-  }]
+  count: 2,
+  objects: [
+    {
+      cluster: [
+        {
+          k8sobj: 'k8sobj',
+          name: 'paas-prod-east2.cluster.k8s.local',
+          objtype: 'cluster',
+          resourceid: 'cluster:paas-prod-east2.cluster.k8s.local',
+          resourceversion: '0',
+          uid: '0x15fa0c'
+        }
+      ],
+      creationtime: '2019-01-24T01:06:47Z',
+      ip: '100.113.249.235',
+      k8sobj: 'k8sobj',
+      labels: '{"app":"cmk-controller","pod-template-hash":"549037671"}',
+      name: 'cmk-controller-prd-qal-98f47cbc5-wf78d',
+      namespace: [
+        {
+          creationtime: '2019-01-17T22:52:00Z',
+          k8sobj: 'k8sobj',
+          labels:
+            '{"iks.intuit.com/owner":"iksm","iks.intuit.com/prune-label":"dev-devx-cmk-controller-use2-prd-paas","name":"dev-devx-cmk-controller-use2-prd-paas"}',
```

Comment:
may want to look thru test data to ensure nothing sensitive

Reply:
agree. will use some dummy ones.